.github/CODEOWNERS

```diff
@@ -68,3 +68,6 @@
 
 # ServiceLabel: %area-BestPractices
 # ServiceOwners:                            @g2vinay @conniey
+
+# ServiceLabel: %area-Monitor-HealthModels
+# ServiceOwners:                        @smritiy
```

Comment:
Recommended to have 2 people here.
You can add me as second person
You will need someone to approve your PRs as per code owners.

Also, add file paths as above services do.